src/GeneralStateTestsFiller/Python/doubleSuicide.py

```diff
@@ -0,0 +1,208 @@
+            storage={},
+        ),
+        addressR: Account(
+            balance=5000000000000000000,
+            nonce=0,
+            code=yul(
+                """
+                    {
+                        let success := """f"""{callR}""""""
+                        mstore(0, 15)
+                        revert(0, 32)
+                    }
+                """
+            ),
+            storage={},
+        ),
+        "0xa94f5374fce5edbc8e2a8697c15331677e6ebf0b": Account(
+            balance=7000000000000000000,
+            nonce=0,
+            code="0x",
+            storage={},
+        ),
+    }
+
+    post = {}
+
+
+    if firstSuicide == "callcode" or firstSuicide == "delegatecall":
+        if is_fork(fork, Cancun):
+            # On Cancun even callcode/delegatecall does not remove the account, so the value remains
+            post["0x095e7baea6a6c7c4c2dfeb977efac326af552d87"] = Account(
+                storage={
+                    "0x01" : "0x01",    # First call to contract S->suicide success
+                    "0x02" : "0x00",    # Second call to contract S->suicide reverted
+                    "0x03" : "15",      # Reverted value to check that revert really worked
+                },
+            )
+        else:
+            # Callcode executed first suicide from sender. sender is deleted
+            post["0x095e7baea6a6c7c4c2dfeb977efac326af552d87"] = Account.NONEXISTENT
+
+        # Original suicide account remains in state
+        post[addressS] = Account(
+            balance=3000000000000000000,
+            storage={}
+        )
+        # Second caller unchanged as call gets reverted
+        post[addressR] = Account(
+            balance=5000000000000000000,
+            storage={}
+        )
+        # Suicide destination
+        post["0x00000000000000000000000000000000000003e8"]: Account (
+            balance=1000000000000000000,
+        )
+
+
+    # On Cancun suicide no longer destroys the account from state, just cleans the balance
+    if firstSuicide == "call":
+        post["0x095e7baea6a6c7c4c2dfeb977efac326af552d87"] = Account(
+            storage={
+                "0x01" : "0x01",    # First call to contract S->suicide success
+                "0x02" : "0x00",    # Second call to contract S->suicide reverted
+                "0x03" : "15",      # Reverted value to check that revert really worked
+            },
+        )
+        # Suicide destination
+        post["0x00000000000000000000000000000000000003e8"]: Account (
+            balance=1000000000000000000,
+        )
+        # Second caller unchanged as call gets reverted
+        post[addressR] = Account(
+            balance=5000000000000000000,
+            storage={}
+        )
+        if is_fork(fork, Cancun):
+            # On Cancun suicide does not remove the account, just sends the balance
+            post[addressS] = Account(
+                balance=0,
+                code="0x6103e8ff",
```

Comment:
If we omit the code in the `post` is ok, it just won't be compared, but if we use python opcodes you can copy the same value we use in the `pre`.

Reply:
Here I think make sense to check only string